app/models/vacancy.rb

```diff
@@ -324,11 +336,8 @@ def delete_documents
 
   private
 
-  def convert_date_to_unix_time(date)
-    # nil.to_i returns 0 in unix time (1970-01-01), so if date is nil we should return nil.
-    return nil if date == nil
-    # Convert to unix time via DateTime object in order to use correct time zone
-    Time.zone.at(date.to_time).to_datetime.midday.to_i
+  def expires_at
+    self.expiry_time.presence || Time.zone.at(self.expires_on.to_time).to_datetime.end_of_day
```

Comment:
So `expiry_time` is a `DateTime` object? I was under the impression it was just a string representation of the time-of-day...

Reply:
`expires_on` is a `Date`, `expiry_time` is a `DateTime` and _almost_ all vacancies will have both fields set. The `expiry_time` is created by adding the time to the `expires_on` date.

Reply:
Eventually we'll want to consolidate those into a single `DateTime` object. When we do we can use temporary attributes on the model the capture the date and expiry time and then consolidate those into a single, stored attribute-or we can use a single `DateTime` picker control and style it to match the design.